installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -14,20 +16,17 @@ vagrant_dev_box_add() {
     # Returns: $box
     box=$1
     declare provider=virtualbox
-    if [[ $_vagrant_dev_host_os == ubuntu ]]; then
```

Comment:
Why is this being removed?

Reply:
I started down the path of removing ubuntu because it is untested for some time now and completely untested for alma. But, then didn't finish. I'll add it back in and we'll just see what happens if we ever use it.